diff --git a/src/background/utils/db.js b/src/background/utils/db.js
index d1ff77f..ba108de 100644
--- a/src/background/utils/db.js
+++ b/src/background/utils/db.js
@@ -31,7 +31,7 @@ import storage, {
 import { storageCacheHas } from './storage-cache';
 import { reloadTabForScript } from './tabs';
 import { vetUrl } from './url';
-import { IS_FIREFOX } from '@/common/safe-globals';
+import { IS_FIREFOX_MV2 } from '@/common/safe-globals';
 
 let maxScriptId = 0;
 let maxScriptPosition = 0;
@@ -216,7 +216,7 @@ export async function initializeDatabase(reset) {
     setInterval(checkRemove, TIMEOUT_24HOURS);
   }
   // declarative userscripts
-  if (IS_FIREFOX && typeof browser.storage.managed === 'object') {
+  if (IS_FIREFOX_MV2 && typeof browser.storage.managed === 'object') {
     try {
       const managedSettings = await browser.storage.managed.get(null);
       if (managedSettings?.options) {
